modules/kmm-replacing-in-tree-modules-with-out-of-tree-modules.adoc

```diff
@@ -21,12 +22,10 @@ spec:
       modprobe:
         moduleName: mod_a
 
-      inTreeModuleToRemove: mod_b
+      inTreeModulesToRemove: [mod_a, mod_b]
 ----
 
-In this example, the `moduleLoader` pod uses `inTreeModuleToRemove` to unload the in-tree `mod_b` before loading `mod_a`
-from the `moduleLoader` image.
-When the `moduleLoader`pod is terminated and `mod_a` is unloaded, `mod_b` is not loaded again.
+In this example, the `moduleLoader` pod uses `inTreeModulesToRemove` to unload the in-tree `mod_b` before loading `mod_a` from the `moduleLoader` image. When the `moduleLoader`pod is terminated and `mod_a` is unloaded, `mod_b` is not loaded again.
```

Comment:
```suggestion
In this example, the `moduleLoader` pod uses `inTreeModulesToRemove` to unload the in-tree `mod_a` and `mod_b` before loading `mod_a` from the `moduleLoader` image. When the `moduleLoader`pod is terminated and `mod_a` is unloaded, `mod_b` is not loaded again.
```

Reply:
Made this change. Any other changes before /lgtm?